src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -393,6 +400,18 @@ internal async void SetSignInStatus(bool status)
             }
         }
 
+        /// <summary>
+        /// Handle the login status changes on splash screen.
+        /// </summary>
+        internal async void HandleSignInStatusChange(bool status)
+        {
+            if (webView?.CoreWebView2 != null)
+            {
+                await webView.CoreWebView2.ExecuteScriptAsync("window.handleSignInStateChange({" +
```

Comment:
maybe nicer with an interpolated string literal, you should not have to do escaping or use "+"

`@$`

https://stackoverflow.com/questions/31638579/how-to-use-verbatim-strings-with-interpolation

Reply:
We still do have to escape `{` with double `{{` and double quotes with a `""`
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim

The other version that also works without verbatim literal:
`$"window.handleSignInStateChange({{\"status\": \"{status}\"}})"`